src/cursor.c

```diff
@@ -432,6 +425,15 @@ process_cursor_motion(struct server *server, uint32_t time)
 	struct cursor_context ctx = get_cursor_context(server);
 	struct seat *seat = &server->seat;
 
+	if (seat->drag.active) {
+		struct drag_icon *icon;
+		double lx = seat->cursor->x;
+		double ly = seat->cursor->y;
+		wl_list_for_each(icon, &seat->drag.icons, link) {
+			wlr_scene_node_set_position(&icon->tree->node, lx, ly);
+		}
+	}
```

Comment:
Not sure if this should be put into a `drag_icons_update()` or similar and thrown into `src/drag_icon.c` instead.

Reply:
removed completely and replaced by setting the single shared drag icon parent